src/settings/SettingsArea.tsx

```diff
@@ -44,7 +50,24 @@ const SettingsArea = () => {
     },
     [settings, setSettings]
   );
-
+  const options = useMemo(() => {
+    const isMac = /Mac/.test(navigator.platform);
+    return {
+      codeStructure: createOptions(
+        codeStructureOptions,
+        "highlight-code-structure",
+        intl
+      ),
+      parameterHelp: createOptions(
+        parameterHelpOptions,
+        "parameter-help",
+        intl,
+        {
+          shortcut: (isMac ? "Cmd" : "Ctrl") + "-Shift+Space",
```

Comment:
```suggestion
          shortcut: (isMac ? "Cmd" : "Ctrl") + "+Shift+Space",
```

Reply:
Fair enough, I think my punctuation is less common (though it's not like you can press them in any order).